servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLSchemaConfiguration.kt

```diff
@@ -159,4 +160,18 @@ class GraphQLSchemaConfiguration {
         contextFactory,
         requestHandler
     )
+
+    @Bean
```

Comment:
Due to the SchemaPrinter's relationship with the Schema I thought this was the most natural place to put this bean, especially given the `SdlRouteConfiguration` also uses the GraphQLSchema, relies on both being injected there. Open to hearing if there's another place the maintainers think is more natural, including either another existing class or a class of its own.